docker-compose.yml

```diff
@@ -30,3 +36,62 @@ services:
             - 27017:27017
         logging:
             driver: none
+
+    mongodb_repl:
+        container_name: mongodb_repl
+        image: mongo:4.2.5
+        restart: always
+        ports:
+            - "27018:27018"
+        entrypoint: [ "/usr/bin/mongod", "--quiet", "--bind_ip_all",  "--port",  "27018", "--replSet", "rs" ]
+        depends_on:
+            - mongodb_repl_2
+            - mongodb_repl_3
+        links:
+            - mongodb_repl_2:mongodb_repl_2
+            - mongodb_repl_3:mongodb_repl_3
+        logging:
+            driver: none
+
+    mongodb_repl_2:
+        container_name: mongodb_repl_2
+        image: mongo:4.2.5
+        restart: always
+        ports:
+            - "27019:27018"
+        entrypoint: [ "/usr/bin/mongod", "--quiet", "--bind_ip_all", "--port",  "27018", "--replSet", "rs" ]
+        depends_on:
+            - mongodb_repl_3
+        logging:
+            driver: none
+
+    mongodb_repl_3:
+        container_name: mongodb_repl_3
+        image: mongo:4.2.5
+        restart: always
+        ports:
+            - "27020:27018"
+        entrypoint: [ "/usr/bin/mongod", "--quiet", "--bind_ip_all",  "--port",  "27018", "--replSet", "rs" ]
+        logging:
+            driver: none
+
+    mongo_repl_init:
+        image: mongo:4.2.5
+        depends_on:
+            - mongodb_repl
+            - mongodb_repl_2
+            - mongodb_repl_3
+        links:
```

Comment:
Delete links please
All containers in same network and can communicate together using service name